docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+sphinx-autobuild
```

Comment:
The docs requirements file already exists at:
https://github.com/mozilla/treeherder/blob/master/requirements/docs.txt